build.gradle

```diff
@@ -178,18 +184,20 @@ try {
     identifiedVersion = versionOverride == null ? gitVersion() : versionOverride
 }
 catch (Exception e) {
-    logger.error("\n\u001B[1;31mThis mod must be version controlled by Git AND the repository must provide at least one tag,\n" +
-                 "or the VERSION override must be set! \u001B[32m(Don't download from GitHub using the ZIP option, instead\n" +
-                 "clone the repository, see\u001B[33m https://gtnh.miraheze.org/wiki/Development \u001B[32mfor details.)\u001B[0m\n");
+    out.style(Style.Failure).text(
+        'This mod must be version controlled by Git AND the repository must provide at least one tag,\n' +
+        'or the VERSION override must be set! ').style(Style.SuccessHeader).text('(Do NOT download from GitHub using the ZIP option, instead\n' + 
+        'clone the repository, see ').style(Style.Info).text('https://gtnh.miraheze.org/wiki/Development').style(Style.SuccessHeader).println(' for details.)'
+    )
     versionOverride = 'NO-GIT-TAG-SET'
     identifiedVersion = versionOverride
 }
 version = minecraftVersion + '-' + identifiedVersion
 String modVersion = identifiedVersion
 
 if( identifiedVersion.equals(versionOverride) ) {
-    logger.error('\u001B[31m\u001B[7mWe hope you know what you\'re doing using\u001B[0m\u001B[1;34m ' + modVersion + '\u001B[0m\n');
-    logger.error('\7\u001B[31mGoing to blindly try to use\u001B[1;34m ' + modVersion + '\u001B[0m\u001B[31m, this probably won\'t work the way you expect!!\u001B[0m\n');
+    out.style(Style.FailureHeader).text('We hope you know what you\'re doing using ').style(Style.Identifier).println(modVersion)
+    out.style(Style.Failure).text('Going to blindly try to override version to ').style(Style.Identifier).text(modVersion).style(Style.Failure).println(', this may not work the way you expect!\7')
```

Comment:
Meh, this warning is annoying.  Just print that you're using a version override :P

Reply:
I can change that, yeah, anyone who downloaded from GitHub with ZIP will have already seen the other